### ISSUE TITLE
Request: in `pcbStands`, add option to have pins on the top instead of the bottom

### ISSUE THREAD

**Issue by DeflateAwning** [mrWheel/YAPP_Box#56]
Currently, you configure which sides (base/lid) have pins and holes with these options:
```
//    n(a) = { <yappBoth> | yappLidOnly | yappBaseOnly }
//    n(b) = { <yappPin>, yappHole } // Baseplate support treatment
```

`yappBoth` puts the pins on the bottom, and the regular connectors on the top.

## Option 1 (preferred):
I'm proposing that the section could instead be setup like this:
```
//    n(a) = { <yappBasePin> | yappBaseHole | yappBaseNoStand } // bottom/base support config
//    n(b) = { <yappLidHole> | yappLidPin | yappLidNoStand } // lid support config
```

This option allows easier configuration of the shape of the PCB stands on each side. The downside is that it's possible to configure it so that there are pins on both sides (which would prevent it from closing). It's easy to avoid doing that though.

## Option 2:
Alternatively, I'm proposing that the line could be changed to:

```
//    n(a) = { <yappBothPinsFromBottom> | yappBothPinsFromTop | yappLidOnly | yappBaseOnly }
//    n(b) would stay roughly the same, I think
```

This change would prevent the user from creating pins on both the top and the bottom, but it feels way less intuitive to me.

**Comment by rosenhauer** [comment]
I think option one makes more sense.  It can be trapped to raise a message if a pin is selected on both.

**Comment by lijon** [comment]
I just came here to ask for the same feature :)

**Comment by rosenhauer** [comment]
Fixed in #62
